build.sbt

```diff
@@ -70,6 +70,7 @@ lazy val plugin = project("sbt-paradox-material-theme", file("plugin"))
     addSbtPlugin("com.lightbend.paradox" % "sbt-paradox" % "0.4.4"),
     libraryDependencies += "org.jsoup" % "jsoup"      % "1.10.3",
     libraryDependencies += "io.circe" %% "circe-core" % "0.9.3",
+    update := update.dependsOn(theme / publishLocal).value,
```

Comment:
So one could argue that a more idiomatic solution for this would be to just use `dependsOn` on the entire project, i.e.

```sbt
lazy val plugin = project("sbt-paradox-material-theme", file("plugin"))
  .enablePlugins(ScriptedPlugin)
  .settings(
    sbtPlugin := true,
    previewSite := {},
    scriptedLaunchOpts += "-Dproject.version=" + version.value,
    scriptedBufferLog := false,
    publishLocal := publishLocal.dependsOn(theme / publishLocal).value,
    addSbtPlugin("com.lightbend.paradox" % "sbt-paradox" % "0.4.4"),
    libraryDependencies += "org.jsoup" % "jsoup"      % "1.10.3",
    libraryDependencies += "io.circe" %% "circe-core" % "0.9.3",
    Compile / resourceGenerators += Def.task {
      val file = (Compile / resourceManaged).value / "paradox-material-theme.properties"
      IO.write(file, s"version=${version.value}")
      Seq(file)
    }
  ).dependsOn(theme)
```
(notice the last line which has `.dependsOn(theme)`).

This however doesn't work since `paradoxTheme` actually resolves the dependency directly rather then pointing to compiled classes in the `target` folder